src/containers/Chat/ChatMessages/ChatMessage/ChatMessage.tsx

```diff
@@ -44,6 +45,7 @@ export const ChatMessage: React.SFC<ChatMessageProps> = (props) => {
   const popperId = open ? 'simple-popper' : undefined;
   let tag: any;
   let deleteId: string | number;
+  const reactStringReplace = require('react-string-replace');
```

Comment:
Shouldn't we import this rather than require?

Reply:
Fixed.